enterprise/enrollment_api.py

```diff
@@ -0,0 +1,39 @@
+# -*- coding: utf-8 -*-
+"""
+Utilities to get details from the course catalog API.
+"""
+from __future__ import absolute_import, unicode_literals
+
+import requests
+from edx_rest_api_client.client import EdxRestApiClient
+
+from django.conf import settings
+
+
+def get_enrollment_api_client():
+    """
+    Retrieve a client of the Enrollment API.
+
+    The client is authenticated using the API token from the Django settings.
+    """
+    session = requests.Session()
+    session.headers = {"X-Edx-Api-Key": settings.EDX_API_KEY}
+    return EdxRestApiClient(
```

Comment:
Do we need to be concerned with exception conditions here?  For example, what if construction fails for some reason?